### PR TITLE
 [CodeQuality] Add SimplifyForeachToCoalescingRector

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

**Comment by TomasVotruba** [comment]
It is. See tests.

I manually fixed similar cases in Rector code recently.

**Comment by carusogabriel** [comment]
> I manually fixed similar cases in Rector code recently.

May you point them to me? I really can see the refactoring :(

If I can reproduce it with http://3v4l.org, there'll be no problem in approving it.

File: packages/CodeQuality/tests/Rector/Foreach_/SimplifyForeachToCoalescingRector/config.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+services:
+    Rector\CodeQuality\Rector\Foreach_\SimplifyForeachToCoalescingRector: ~
```
Comment (carusogabriel) [inline packages/CodeQuality/tests/Rector/Foreach_/SimplifyForeachToCoalescingRector/config.yml]: This should also be added to `code-quality.yml` level

Comment (TomasVotruba) [reply packages/CodeQuality/tests/Rector/Foreach_/SimplifyForeachToCoalescingRector/config.yml]: I'll handle it with autoregistration of whole package :+1:

**Comment by TomasVotruba** [comment]
> May you point them to me? I really can see the refactoring :(

I have no exact commit to show :) 

Try last tag to master: https://github.com/rectorphp/rector/compare/v0.3.13...master

**Comment by TomasVotruba** [comment]
> If I can reproduce it with http://3v4l.org, there'll be no problem in approving it.

What problem?

**Pull request merged**